addons/auth_signup/views/auth_signup_login_templates.xml

```diff
@@ -1,36 +1,38 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
         <template id="auth_signup.login" inherit_id="web.login" name="Sign up - Reset Password">
-            <xpath expr="//button[@type='submit']" position="before">
-                <a t-if="signup_enabled" t-attf-href="/web/signup?{{ keep_query() }}" class="btn btn-link float-right" role="button">Don't have an account?</a>
-                <a t-if="reset_password_enabled" t-attf-href="/web/reset_password?{{ keep_query() }}" class="btn btn-link float-right" role="button">Reset Password</a>
+            <xpath expr="//button[@type='submit']" position="after">
+                <div class="justify-content-between mt-2 no-gutters row small">
```

Comment:
A `.row.no-gutters` without any column inside is probably not what was meant to be used here. `d-flex` ? `btn-group` ?

Reply:
Right! Thanks!